Research/ResearchUI/iOS/RSDStepViewController.swift

```diff
@@ -1052,7 +1057,7 @@ open class RSDStepViewController : UIViewController, RSDStepController, RSDCance
             
             // Update the countdown
             if let stepDuration = self.activeStep?.duration {
-                countdown = Int(stepDuration - duration)
+                countdown = Int(stepDuration) - Int(floor(duration))
```

Comment:
unless duration can be negative, I think the floor() is redundant? maybe worth keeping for explicitness, but then stepDuration should be handled the same way